src/json-ld.net/Util/JSONUtils.cs

```diff
@@ -130,6 +121,11 @@ public static string ToString(JToken obj)
             return sw.ToString();
         }
 
+        public static JToken FromURL(Uri url)
+        {
+            return FromURLAsync(url).GetAwaiter().GetResult();
+        }
+
```

Comment:
Added async and sync variants for the two methods that I enlightened with `HttpClient` to cover all of our supported platforms.